### PR TITLE
bare-metal-os module doc

### PR DESCRIPTION
**Pull request by adean20csc**
This is the doc for the bare-metal-os module. It is replacing the previous bare-metal-os.rst.

It also renames the file in the index.

### CODE REVIEW

File: docsite/source/bare_metal_os_module.rst [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+group_vars
+~~~~~~~~~~
+
+As mentioned above Ansible allows you to group hosts and assign variables to a
+:code:`[group]`. This allows you to run plays against multiple hosts without
+having to specify them individually. The group_vars variables are in the
+``bare-metal-os/inventory/group_vars/`` folder. The bare_metal_os.yml is used by
+all hosts in the hosts.ini.
+
```
Comment (jdextercsc) [inline docsite/source/bare_metal_os_module.rst]: I Think you need information about the bonds/vlans in this section.

**Comment by mtnbikenc** [comment]
Review 👍

**Comment by mtnbikenc** [comment]
Review 👍

**Comment by jcpowermac** [comment]
Review 👍

**Pull request merged**